### PR TITLE
Minor updates to SplitTest docs

### PR DESCRIPTION
**Pull request by eessex**
This does not include all items discussed with @xtina-starr for improving these docs, but adds a few of them:
- Add an intro explaining basically what this component does and steps to start a test
- Break up the structure a bit more
- Convert some elements to JS from coffee
- Move and elaborate on section about `splitTest.view()`

### CODE REVIEW

**Comment by eessex** [comment]
Adds a few steps we discussed, will need to do a bit more research to get more in depth on outcomes.

File: src/desktop/components/split_test/README.md [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
+The Split Test component enables the bucketing of users into categories. When a test is enabled, a cookie is set for each user that is also stored on the window via sharify. The cookie/SD var's name is based on the test name, and its value is set to one of the test's possible outcomes. Engineers can expose multiple variations of features, content, components etc. by hooking into a test outcome.
+
+Rolling out a test requires a few steps:
+
+- Add a test with name and outcomes to `running_tests.coffee`
+- Create any feature variations that should be exposed based on outcome
+- Configure analytics events that will be tracked as part of the test
+- Enable the test via a `splitTest.view()` event
+
+The last step is especially important! Users will still receive cookies, and analytics will operate as expected with or without a `splitTest.view()` event, but it is this call that enables the data team to parse analytics in the context of a test.
```
Comment (ashfurrow) [inline src/desktop/components/split_test/README.md]: Great job calling this out!

Comment (xtina-starr) [reply src/desktop/components/split_test/README.md]: I add some code snippets in my [PR](https://github.com/artsy/force/pull/3776). We are going to work to consolidate.

**Comment by eessex** [comment]
Updated this with @xtina-starr to combine with efforts in https://github.com/artsy/force/pull/3776

**Comment by xtina-starr** [comment]
🎉

**Pull request merged**